dbms/src/Functions/FunctionsLogical.cpp

```diff
@@ -275,11 +275,39 @@ class AssociativeGenericApplierImpl<Op, 1>
 };
 
 
+template <typename Op>
+class AssociativeGenericRuntimeApplierImpl
+{
+    using ResultValueType = typename Op::ResultType;
+
+public:
+    AssociativeGenericRuntimeApplierImpl(const ColumnRawPtrs & _in)
+        : in{_in} {}
+
+    inline ResultValueType apply(const size_t i, const size_t column_counter) const
+    {
+        const auto a = ValueGetterBuilder::build(in[in.size() - column_counter])(i);
```

Comment:
Unfortunately this line will, for one, ruin any attempts of vectorized execution optimizations